addons/mail/static/src/components/chatter_topbar/chatter_topbar_tests.js

```diff
@@ -437,27 +419,28 @@ QUnit.test('rendering with multiple partner followers', async function (assert)
     assert.expect(7);
 
     await this.start();
-    this.data['res.partner'].records = [{
+    this.data['res.partner'].records.push({
         id: 100,
         message_follower_ids: [1, 2],
-    }];
-    this.data['mail.followers'].records = [
+    });
+    this.data['mail.followers'].records.push(
         {
             // simulate real return from RPC
             // (the presence of the key and the falsy value need to be handled correctly)
             channel_id: false,
             id: 1,
             name: "Jean Michang",
             partner_id: 12,
-        }, {
+        },
+        {
```

Comment:
I think convention is `}, {` like before, but honestly I don't care much.

Reply:
This convention is bad, because it goes against the convention that each element of a multi-line array/object should be on its own line.

It's also bad because that means there is no indent difference between the close of the parent array and each of its element.

Reply:
It also makes copy/pasting elements harder, and it's inconsistent because the last element doesn't have the ending `,`.